app/helpers/webhooks_helper.rb

```diff
@@ -1,11 +1,23 @@
 # frozen_string_literal: true
 module WebhooksHelper
+  GENERIC_SOURCES = [
+    ['Any CI', 'any_ci'],
+    ['Any code push', 'any_code'],
+    ['Any Pull Request', 'any_pull_request'],
+    ['Any', 'any']
+  ].freeze
+
+  NO_SOURCE = ['None', 'none'].freeze
+
   def webhook_sources(sources)
-    [
-      ['Any CI', 'any_ci'],
-      ['Any code push', 'any_code'],
-      ['Any Pull Request', 'any_pull_request'],
-      ['Any', 'any']
-    ] + sources.map { |source| [source.titleize, source] }.to_a
+    titleize_sources(sources) + GENERIC_SOURCES
+  end
+
+  def webhook_sources_select_option(sources)
+    titleize_sources(sources) + GENERIC_SOURCES << NO_SOURCE
+  end
+
+  def titleize_sources(sources)
```

Comment:
hmmm idk this is getting weird ... only have a single webhook_sources_for_select method that takes a `none: true|false` option ?